frontend/src/components/reviewed-cars/reviewed-cars.tsx

```diff
@@ -0,0 +1,26 @@
+import car from '@assets/images/mock_car_picture.png';
+import { ShortCarCard } from '@components/short-car-card/short-car-card';
+
+import styles from './styles.module.scss';
+
+const ReviewedCars = (): JSX.Element => {
+  // mock data
+  const carData = {
+    src: car,
+    title: 'BMW X5 M Packet 40d 2015',
+    price: '$ 34 000 - 52 450',
+  };
```

Comment:
We should use `photo_url` instead of `src` to be closer to the DB

Reply:
I fixed that